cmd/kubeadm/app/cmd/init.go

```diff
@@ -110,6 +110,8 @@ type initOptions struct {
 	dryRun                bool
 	kubeconfigDir         string
 	kubeconfigPath        string
+	csrOnly               bool
```

Comment:
These don't really belong to init... The option should be plumbed down lower as the option only applies to certs.

Reply:
we already it this way for all phases.
my suggestion would be to go with this for 1.13 and think of eventual refactor later.